### PR TITLE
Building floor views instead of map

### PR DESCRIPTION
**Pull request by sgurnoor**

### CODE REVIEW

File: MHacks/Assets.xcassets/image1.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  }
+}
```
Comment (sgurnoor) [inline MHacks/Assets.xcassets/image1.imageset/Contents.json:21]: placeholder images! waiting for the final designs

**Review by sgurnoor — COMMENTED** [review]
Haven't purged any of the map view controller stuff

File: MHacks/BuildingMapsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+//  Created by Gurnoor Singh on 9/16/16.
+//  Copyright © 2016 MHacks. All rights reserved.
+//
+
+import UIKit
+
+class BuildingMapsViewController: UIViewController, UIScrollViewDelegate {
+    let colors = [UIColor.red, UIColor.blue, UIColor.green, UIColor.yellow]
+    var frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+    @IBOutlet weak var scrollView: UIScrollView!
```
Comment (manavgabhawala) [inline MHacks/BuildingMapsViewController.swift:14]: Is there a good reason we aren't using a UIPageViewController instead?

Comment (russellladd) [reply MHacks/BuildingMapsViewController.swift:14]: It's too heavy. Each page doesn't need to be its own view controller.

**Pull request merged**